test/mocks/request_info/mocks.h

```diff
@@ -66,14 +71,16 @@ class MockRequestInfo : public RequestInfo {
   absl::optional<std::chrono::nanoseconds> first_downstream_tx_byte_sent_;
   absl::optional<std::chrono::nanoseconds> last_downstream_tx_byte_sent_;
   absl::optional<std::chrono::nanoseconds> end_time_;
-  Network::Address::InstanceConstSharedPtr upstream_local_address_;
-  Network::Address::InstanceConstSharedPtr downstream_local_address_;
-  Network::Address::InstanceConstSharedPtr downstream_remote_address_;
   absl::optional<Http::Protocol> protocol_;
   absl::optional<uint32_t> response_code_;
+  envoy::api::v2::core::Metadata metadata_;
+
+private:
```

Comment:
IIUC, it's not typical in Envoy to make member variables in a mock `private` since it restricts how the mock can be used by the tests, and there is no need to enforce modularity or restrictive interfaces for test-only classes.  Is there a reason these fields need to be `private`?  For instance, one may wish to set `bytes_sent_` to `12` and then modify it to `7` later in the test.  This is not possible with the new interface.  (I'm commenting now because this new restriction caused a minor hangup in our weekly import :) )